libs/blocks/icon-block/icon-block.css

```diff
@@ -172,10 +174,14 @@
   margin-bottom: 0;
 }
 
-.icon-block.vertical:not(.center) .foreground .text-content .action-area > a {
+.icon-block.vertical:not(.center) .foreground .text-content .action-area > a,
+.icon-block.inline:not(.center) .foreground .text-content .action-area > a {
   margin-right: var(--spacing-s);
   white-space: nowrap;
 }
+.icon-block.inline:not(.center) .foreground .text-content .action-area > a:first-child {
+  margin-bottom: var(--spacing-xxs);
+}
 
```

Comment:
Rather than targeting the <a> tags margin(s). 
Can you try adding some styles to the .action-area using flex with a gap? 
```
.icon-block .foreground .text-content .action-area {
    display: flex;
    gap: var(--spacing-s);
}
'''
If we were to approach it at the inline block level it would be good to use the 'margin-block' properties for better localization text-orientation support. Just a thought on that.

Reply:
```
  column-gap: var(--spacing-s);
  row-gap: var(--spacing-xs);
```
I used flex and this gaps instead of margins. Is this what you meant ?